File: Backend/src/main/java/com/feedle/feedleapi/Models/FriendRequestNotification.java
package com.feedle.feedleapi.Models;

import lombok.Data;

@Data
public class FriendRequestNotification {
    public int friendRequestId;
    public int userId;
    public int creatorId;
    public int potentialFriendUserId;
    public String content;
    public String potentialFriendUserName;
    public String creatorUserName;
}

File: Front-end/Feedle/Models/FriendRequestNotification.cs
using System.Text.Json.Serialization;

namespace Feedle.Models
{
    public class FriendRequestNotification
    {
        [JsonPropertyName("friendRequestId")]
        public int FriendRequestId { get; set; }
        
        [JsonPropertyName("creatorId")]
        public int CreatorId { get; set; }
        
        [JsonPropertyName("userId")] public int UserId { get; set; }
        [JsonPropertyName("potentialFriendUserId")] 
        public int PotentialFriendUserId { get; set; }
        [JsonPropertyName("creatorUserName")]
        public string CreatorUserName { get; set; }
        [JsonPropertyName("content")]
        public string Content { get; set; }
        [JsonPropertyName("potentialFriendUserName")]
        public string PotentialFriendUserName { get; set; }
    }
}